Xbim.Geometry.Regression/BatchProcessor.cs

```diff
@@ -169,11 +169,86 @@ private ProcessResult ProcessFile(string ifcFile, StreamWriter writer, ILogger<B
                             };
 
                         }
+
+                        // Option to save breps of encountered classes by type or entityLabel for debugging purposes
+
+                        if (_params.WriteBreps)
+                        {
+                            var path = Path.Combine(
+                                    Path.GetDirectoryName(ifcFile),
+                                    Path.GetFileName(ifcFile) + ".brep.unclassified");
+                            IXbimGeometryEngine engine = new XbimGeometryEngine();
+                            if (!Directory.Exists(path))
+                                Directory.CreateDirectory(path);
+                            IfcStore s = model as IfcStore;
+                            if (s != null)
+                            {
+                                var ents = new List<IPersistEntity>();
+
+                                var exportBrepByType = new string[]
+                                {
+                                    "IfcFacetedBrep",
+								    // IIfcGeometricRepresentationItem
+								    "IfcCsgSolid",
+                                    "IfcExtrudedAreaSolid",
+                                    "IfcExtrudedAreaSolidTapered",
+                                    "IfcFixedReferenceSweptAreaSolid",
+                                    "IfcRevolvedAreaSolid",
+                                    "IfcRevolvedAreaSolidTapered",
+                                    "IfcSurfaceCurveSweptAreaSolid",
+                                    "IfcSectionedSolidHorizontal",
+                                    "IfcSweptDiskSolid",
+                                    "IfcSweptDiskSolidPolygonal",
+                                    "IfcBooleanResult",
+                                    "IfcBooleanClippingResult",
+								    // composing objects
+								    "IfcConnectedFaceSet"
```

Comment:
Inconsitent formatting in this block, it's sometimes using spaces and sometimes tabs. That makes the code not well aligned.